test/integration/query-interface.test.js

```diff
@@ -190,7 +208,13 @@ describe(Support.getTestDialectTeaser('QueryInterface'), function() {
           }
           expect(isAdmin.type).to.equal(assertVal);
           expect(isAdmin.allowNull).to.be.true;
-          expect(isAdmin.defaultValue).to.be.null;
+          switch (dialect) {
+            case 'sqlite':
+              expect(isAdmin.defaultValue).to.be.undefined;
```

Comment:
Should it be? I haven't explored this behavior with other dialects and I don't have tests set up to run other dialects' tests, so I don't really feel comfortable making changes for other dialects other than sqlite.